readme.txt

```diff
@@ -301,6 +303,9 @@ Yes, you will need to know the key name of the field. Then include an array in y
 
 == Changelog ==
 
+= 3.15.1 =
+* Create/modified shortcodes to filter meetings by passed types. [more info]
```

Comment:
missing link for more info

should be `3.15.2`

Reply:
Thx. Will fix...